### ISSUE TITLE
Hard to follow breaking changes

### ISSUE THREAD

**Issue by BYODKM** [necolas/normalize.css#560]
Each of major update of normalize.css  is a little challenge for me to understand which element has a breaking change.

So, I've sent PR #559 like this.

```
/* Forms
   ========================================================================== */

button {
  /* Change font properties to `inherit` in all browsers (opinionated). */
  font: inherit;

  /* Show the overflow in IE. */
  overflow: visible;

  /* Remove the margin in Safari. */
  margin: 0;

  /* Remove the inheritence of text transform in Edge, Firefox, and IE. */
  text-transform: none;

  /* Change the cursor in all browsers (opinionated). */
  cursor: pointer;

  /* Correct the inability to style clickable types in iOS. */
  -webkit-appearance: button;
}

button::-moz-focus-inner {
  /* Remove the inner border and padding in Firefox. */
  border: 0;
  padding: 0;
}

button:-moz-focusring {
  /* Restore the focus styles unset by the previous rule. */
  outline: 1px dotted ButtonText;
}

[disabled] {
  /* Restore the default cursor to disabled elements unset by the previous rule. */
  cursor: default;
}

fieldset {
  /* Change the border, margin, and padding in all browsers (opinionated). */
  border: 1px solid #c0c0c0;
  margin: 0 2px;
  padding: 0.35em 0.625em 0.75em;
}

input {
  /* Change font properties to `inherit` in all browsers (opinionated). */
  font: inherit;

  /* Show the overflow in Edge and IE.*/
  overflow: visible;

  /* Remove the margin in Safari. */
  margin: 0;
}

input::-moz-focus-inner {
  /* Remove the inner border and padding in Firefox. */
  border: 0;
  padding: 0;
}

input:-moz-focusring {
  /* Restore the focus styles unset by the previous rule. */
  outline: 1px dotted ButtonText;
}
```

Please consider this.

**Comment by battaglr** [comment]
@BYODKM, hi! When I first saw this kind of comments I found it very weird to see references like `/* 1 */` or `/* 2 */`. Today I use that kind of comments in a daily basis.

The great benefit for me is that you can easily go through the code without the comments being in your way; but if you think you're missing something of need more info, you can easily refer to them —it's like "unobtrusive comments". Of course, this is a matter of taste and style, and there's no right or wrong.

**Comment by BYODKM** [comment]
@battaglr, Sorry, my point is it would be nice if each elements appeared only once in normalize.css, that is easy to understand whitch element has been modified. No need to scroll around.
I know it would be a little mess about comments, but I believe it's worth to maintain the code more easily.

**Comment by jonathantneal** [comment]
First, thank you for doing this. I wish it were something I could pull in if only because you’ve made a really clean PR and opened an issue which explains your point of view very clearly. This means a lot to me, and almost any maintainer, so thank you.

We did some re-organizing in this major release, and we wrote about our coding conventions in [CONTRIBUTING.md](https://github.com/necolas/normalize.css/blob/master/CONTRIBUTING.md#css-conventions).

I think you are onto something with individual elements, but I would actually like to see it go a bit further. I imagine something like a lib directory with each normalization in a separate file paired with an individual test file. I’m closing this only because it does not meet our already decided upon conventions for normalize v4. If we change things, I would prefer to move in the modular direction. Though I’m closing it, I would still really like to know what you think.
